modules/Core/pages/panel/general_settings.php

```diff
@@ -245,8 +234,8 @@
 }
 
 // Get form values
-$contact_email = $queries->getWhere('settings', ['name', 'incoming_email']);
-$contact_email = Output::getClean($contact_email[0]->value);
+$incoming_email = $queries->getWhere('settings', ['name', 'incoming_email']);
```

Comment:
Why you even keep this when you removed the setting? from my understanding the setting should still exist as incoming email